implementations/elixir/ockam/ockam_healthcheck/lib/application.ex

```diff
@@ -19,32 +20,58 @@ defmodule Ockam.Healthcheck.Application do
     case Jason.decode(config_json) do
       {:ok, targets} when is_list(targets) ->
         Enum.reduce(targets, {:ok, []}, fn
-          target, {:ok, targets} ->
-            case target do
-              %{
-                "name" => name,
-                "host" => host,
-                "port" => port
-              }
-              when is_integer(port) ->
-                api_worker = Map.get(target, "api_worker", "api")
-                healthcheck_worker = Map.get(target, "healthcheck_worker", "healthcheck")
+          %{
+            "name" => name,
+            "host" => host,
+            "port" => port,
+            "path" => path,
+            "method" => method,
+            "crontab" => crontab,
+            "healthcheck_worker" => healthcheck_worker
+          } = target,
+          {:ok, targets}
+          when is_integer(port) ->
+            api_worker = Map.get(target, "api_worker", "api")
+            body = target |> Map.get("body") |> decode_body()
+            method = String.to_existing_atom(method)
```

Comment:
I agree `to_existing_atom` is ultimately the right call here, but we may want to somehow emphasize or caution about this in the README?

Reply:
Added to readme in https://github.com/build-trust/ockam/commit/a23dafbc12b9fb12be1dcebe2a75921eb9decc59

Reply:
Super clear, thanks!